tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2228,6 +2007,40 @@ public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $
 		);
 	}
 
+	/**
+	 * @Then /^the JSON data of the response should (not )?contain the user "([^"]*)" in the item 'value'(?:, the user-details should match)?$/
+	 * @Then /^the JSON data of the response should (not )?contain the group "([^"]*)" in the item 'value'(?:, the group-details should match)?$/
+	 *
+	 * @param string $shouldOrNot (not| )
+	 * @param string $user
+	 * @param PyStringNode|null $schemaString
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theJsonDataResponseShouldContainDisplayNameAndMatch(
+		string $shouldOrNot,
+		string $user,
+		?PyStringNode $schemaString = null
+	): void {
+		$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent()->value;
+		$displayNameFound = false;
+		foreach ($responseBody as $value) {
+			if (isset($value->displayName) && $value->displayName === $user) {
+				$responseBody = $value;
+				$displayNameFound = true;
+				break;
+			}
+		}
+		if ($shouldOrNot === 'not ' && !$displayNameFound) {
+			return;
```

Comment:
perhaps an assert should be added to display an error message if the user or group exists